src/main/java/org/edumips64/utils/cli/ShowCommand.java

```diff
@@ -0,0 +1,46 @@
+package org.edumips64.utils.cli;
+
+import picocli.CommandLine.Command;
+import picocli.CommandLine.ParentCommand;
+import picocli.CommandLine.Parameters;
+import picocli.CommandLine.Option;
+
+@Command(name = "show", resourceBundle = "CliMessages")
+public class ShowCommand implements Runnable{
+
+    @ParentCommand
+    private Cli cli;
+
+    @Option(names={"-h","--help"}, usageHelp = true, descriptionKey= ".help.usage.description")
+    private boolean help;
+
+    @Command(name="registers")
+    private void showRegisters() {
```

Comment:
Would it be possible to add commands to see the FP and special registers (LO/HI/FCSR)?

Reply:
Added in the most recent commit, incoming here shortly.

Reply:
I just tried it out, and it work.

I am ok with submitting as it is, but I have a few suggestion for a future iteration:
- the `show register` command could be the entry point to see all registers, including single FP registers and special registers. It'd be nice to support all the names supported by the parser (e.g. `show register r1`, `show register f12`, `show register lo`) including aliases (`show register $a0`, etc...)

Again, it's ok as it is now, just writing things down for future improvements!